### PR TITLE
readme: info on dev server over local network

### PR DESCRIPTION
**Pull request by meows**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,15 +29,19 @@
 
 # Running DB and App
 
-1. Start dev server in another terminal window then nagivate in a new browser tab to `localhost:3000`.
+1. Start dev server in another terminal window then nagivate in a new browser
```
Comment (hamirmahal) [inline README.md:32]:
```suggestion
1. Start the dev server in another terminal window, then navigate in a new browser
```

Comment (hirehamir) [inline README.md]:
```suggestion
1. Start the dev server in another terminal window, then navigate in a new browser
```

This removes a trailing space, and fixes a typo with the word `navigate`.

Comment (meows) [reply README.md]: thx fixed

Comment (hirehamir) [reply README.md]: You're welcome! Thanks for accepting the suggestion.

**Pull request merged**